airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_inline_schemas/pipeline.py

```diff
@@ -198,7 +198,7 @@ async def _run(self) -> StepResult:
             _update_inline_schema(schema_loader, json_streams, stream_name)
 
         write_yaml(data, manifest_path)
-        await format_prettier([manifest_path], logger=logger)
+        # await format_prettier([manifest_path], logger=logger)
```

Comment:
Commented out the formatting as it was causing the command to hang regardless of success or failure.

Reply:
That's tolerable if you just do `format fix all` separately before pushing up PRs. Single responsibility principle!